metadata-io/src/main/java/com/linkedin/metadata/search/ranker/SearchRanker.java

```diff
@@ -52,9 +58,9 @@
   /**
    * Add the extracted features into each search entity to return the features in the response
    */
+  @SneakyThrows
   private SearchEntity updateFeatures(SearchEntity originalEntity, Features features) {
-    return new SearchEntity().setEntity(originalEntity.getEntity())
-        .setMatchedFields(originalEntity.getMatchedFields())
+    return originalEntity.clone()
```

Comment:
fancy

Reply:
I got mad having this build fail every time I have to add a new field :(